### DISCUSSION TITLE
Set the minimum signal strength for FM DX

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#578] [Ideas]
The new FM DX feature is great.  I would just like to be able to set a minimum signal level to stop scanning.  So far it still ends up with a lot of false detections where there is actually no useful signal.  The user could optionally set the signal level at which the scan stops.

**Comment by PE5PVB** [comment]
You can adjust the sensitivity in FM settings menu -> Scan sensitivity.

**Comment by Mike86-CZE** [comment]
I didn't think of that.  Thanks
